docs/reference-architectures/serverless/web-app.md

```diff
@@ -144,6 +131,8 @@ In this architecture, the client application is a single-page application (SPA)
 
 To configure authentication:
 
+Why not use Easy Auth here on the Function app? 
```

Comment:
Easy Auth is built in to Azure Fuctions which takes care of all of the session handling. It also allows developers to use social auth in addition to Active Directory.

We set this up for theurlist.com by adding a rule in Front Door to route all .auth traffic to the function app. Some header configuration in Front Door has to be tweaked to do this. Also, the domain needs to be assigned in the function app, and then removed so the function app sends the right request. We need to fix that with the Front Door team. Perhaps AD is a better solution for now just based on that.